src/executables/unittests_cqueue.c

```diff
@@ -27,14 +27,17 @@
 #include <pthread.h>
```

Comment:
Remove unittests_cqueue.c. and the unit test. New version of lock-free is already tested with new unit test system :).